solidity/test/utils/DSTestFull.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.4 <0.9.0;
 
-import 'forge-std/console2.sol';
-import 'prb/test/PRBTest.sol';
+import {console} from 'forge-std/console.sol';
```

Comment:
@eugenioclrc could you elaborate why console2 is not necessary with `PRBTest`?

Reply:
Its not because `PRBTest`, tbh i have run a diff between `console.sol` and `console2.sol` and the only difference is that **console.sol** uses `uint` and **console2.sol** `uint256`

To keep it simple, just use console.sol